charts/scalardl/templates/ledger/deployment.yaml

```diff
@@ -27,12 +27,24 @@ spec:
     {{- end }}
       securityContext:
         {{- toYaml .Values.ledger.podSecurityContext | nindent 8 }}
+      volumes:
+      {{- if .Values.ledger.scalarLedgerConfiguration.ledgerProofEnabled }}
+        - name: "{{ .Values.ledger.scalarLedgerConfiguration.secretName }}"
+          secret:
+            secretName: "{{ .Values.ledger.scalarLedgerConfiguration.secretName }}"
+      {{- end }}
       containers:
         - name: {{ .Chart.Name }}-ledger
           securityContext:
             {{- toYaml .Values.ledger.securityContext | nindent 12 }}
           image: "{{ .Values.ledger.image.repository }}:{{ .Values.ledger.image.version }}"
           imagePullPolicy: {{ .Values.ledger.image.pullPolicy }}
+          volumeMounts:
+          {{- if .Values.ledger.scalarLedgerConfiguration.ledgerProofEnabled }}
+            - name: "{{ .Values.ledger.scalarLedgerConfiguration.secretName }}"
+              mountPath: "/{{ .Values.ledger.scalarLedgerConfiguration.secretName }}"
```

Comment:
Making here a variable will make the template a bit more complicated. This is fine as it is, but if the fixed path is not a problem, you may fix it.

Reply:
Thanks!
Fixed path (secret name) is technically fine and it was actually a fixed path like `/key-pair`.
But if it is fixed, I think users have no idea about the path name unless they look at the deployment.yaml or we document the name manually in other than README, which I think is not user-friendly or developer-friendly.
To mitigate the minor issue, I changed it to a variable, so that it is documented in the README automatically with the default value and users can see what to specify for the secret name.

But I wasn't sure if it is the way.
Is there any better way?
Or should it be a fixed path and we should document what secret name to specify somewhere else manually?

Reply:
Fixed in 3b3076c12d423114bd23121f97a98d942b3227e. PTAL again! :bow: